packages/components/src/components/tables/es-table/es-table.tsx

```diff
@@ -177,7 +182,14 @@ export class Table {
                             cellCount: cells.length,
                         })}
                     >
-                        {cell.cell
+                        {this.loading &&
+                            cell.loading?.(h, {
+                                ...(this.extraCellProps?.(key, data) ?? {}),
+                                key,
+                                data,
+                                parent: this.identifier,
+                            })}
```

Comment:
Do we want a default loading, so we don't need to pass it everywhere?

```tsx
<es-loading-text expectedLength={cell.title?.length ?? 30} />
```

Would probably be a sensible default. 

Or we could add `expectedLength` to the `TableCell` definition and have:

```tsx
<es-loading-text expectedLength={cell.expectedLength ?? cell.title?.length ?? 30} />
```

So now a table definition can be as before:

```tsx
name: { title: 'User Name' },
```

If it has a known expected length:
```tsx
id: { 
    title: 'Project Id',
    expectedLength: 20,
},
```

If you want to do something fancy:
```tsx
fancy: { 
    title: 'Something fancy',
    loading: () => "I am loading!",
},
```

We could then disable a loading cell as needed with `() => null`